examples/nomad-consul-separate-cluster/variables.tf

```diff
@@ -3,8 +3,7 @@
 # Define these secrets as environment variables
 # ---------------------------------------------------------------------------------------------------------------------
 
-# AWS_ACCESS_KEY_ID
-# AWS_SECRET_ACCESS_KEY
+# None
```

Comment:
Why remove the env vars? This is a safer way to manage AWS creds than storing them in plain text, so I'd prefer to leave those here.

Reply:
With the usage of AWS profiles this mechanism is not mandatory anymore. AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY do not have to be given to run terraform. The AWS profile or the default profile is chosen for AWS access.